odoo/tools/config.py

```diff
@@ -200,8 +200,10 @@ def _build_cli(self):
                          help="install one or more modules (comma-separated list, use \"all\" for all modules), requires -d")
         group.add_option("-u", "--update", dest="update", type='comma',  metavar="MODULE,...", my_default=[], file_loadable=False,
                          help="update one or more modules (comma-separated list, use \"all\" for all modules). Requires -d.")
-        group.add_option("--without-demo", dest="without_demo", my_default=False, type='without_demo', metavar='BOOL', nargs='?', const=True,
-                         help="use with -i/--init, skip installing fake demonstration data (e.g. Mitchel Admin/Azure Interior)")
+        group.add_option("--with-demo", dest="with_demo", action='store_true', my_default=False,
+                         help="use with -i/--init, enable installing fake demonstration data (e.g. Mitchel Admin/Azure Interior)")
+        group.add_option("--without-demo", dest="with_demo", type='without_demo', metavar='BOOL', nargs='?', const=True,
+                         help="use with -i/--init, disable installing fake demonstration data (e.g. Mitchel Admin/Azure Interior)")
```

Comment:
Maybe we can make the help more accurate
in current master the --without-demo means something like 
`skip installing demo data for new initialized database by default`
which means when `--without-demo`
1. for database already installed with demo data, if you install more modules, those new modules will have demo data
2. if you create a new database with specification `with demo` (e.g. using database manager), new modules will have demo data.